### ISSUE TITLE
unstarring npm from npmjs.com or cli impossible

### ISSUE THREAD

**Issue by jwldr** [npm/registry-issue-archive#88]
Hello,

upon clicking on the star at https://www.npmjs.com/package/npm it seems to disappear, but when I reload the page the star is present again on the npm package.

So I opened the console in my browser and realized there was an POST error in index.min.js on line 7776: s.send(t.hasContent && t.data || null ) -> Failed to load resource: the server responded with a status of 500 (Internal Server Error)

My location is Germany and login works normally (cli & website),
npm unstar/star other packages works normally (using npm cli or website)

`npm unstar npm` also returns an error, for which I attached the [npm-debug.log](https://gist.github.com/logol/3ec53c8ed36e666a595e874551a5990a) as gist.

If you need more details please let me know.

PS: I wrote to support@npmjs.com before (request 10457) and they asked me to open an issue here.

**Comment by CedX** [comment]
Duplicate of #79 ?

**Comment by soldair** [comment]
@cedx this is not a duplicate. 

@logol 
we have logic of uncertain age that prevents changes to stars on the `npm` package itself.
we believe this was a measure to prevent the npm package metadata from growing and impacting performance. We discussed a fix internally but have no immediate plans. 

There is certainly no reason i can think of for blocking `unstar` requests. so perhaps we just fix that for now.

**Comment by jwldr** [comment]
@soldair 
Thanks for the feedback, unblocking unstar requests would be nice because I like to use `npm stars` to install commonly used packages (similar to the `apm stars --install` command) and by now I always need to sed out npm

**Comment by soldair** [comment]
Oh interesting use case. Ill take a look
